pkg/commands/git_commands/commit.go

```diff
@@ -199,8 +199,11 @@ func (self *CommitCommands) AmendHeadCmdObj() oscommands.ICmdObj {
 func (self *CommitCommands) ShowCmdObj(sha string, filterPath string, ignoreWhitespace bool) oscommands.ICmdObj {
 	contextSize := self.UserConfig.Git.DiffContextSize
 
+	extDiffCmd := self.UserConfig.Git.Paging.ExternalDiffCommand
 	cmdArgs := NewGitCmd("show").
+		ConfigIf(extDiffCmd != "", "diff.external="+extDiffCmd).
```

Comment:
```suggestion
		ConfigIf(extDiffCmd != "", "diff.external="+extDiffCmd).
		ArgIfElse(extDiffCmd != "", "--ext-diff", "--no-ext-diff").
		Arg("--submodule").
```
A little clearer imo (aligning the two calls at the cost of a double negative)

Reply:
Makes sense; changed in 9121114c5934 (together with the other ones below).